axelrod/player.py

```diff
@@ -52,6 +53,12 @@ def update_history(player, move):
         player.defections += 1
 
 
+def update_play_counts(player, move, reply):
```

Comment:
In the glossary <http://axelrod.readthedocs.io/en/latest/reference/glossary.html#a-play> we have defined a play as "a single player choosing an action."

In the results set and the interaction utils we have called these counters "state distributions". Could we call them the same thing here (for consistency).

- `update_state_distribution`
- `player_state_distribution`